### ISSUE TITLE
Training not working in calamari 2.3.1

### ISSUE THREAD

**Issue by abhikatoldtrafford** [Calamari-OCR/calamari#275]
My old calamari train command is:
CUDA_VISIBLE_DEVICES=0 calamari-train --n_augmentations .1 --num_threads 8 --batch_size 16 --network=cnn=64:3x3,pool=2x2,cnn=128:3x3,pool=2x2,lstm=300,dropout=0.1,learning_rate=.00001 --epochs 15 --checkpoint_frequency 1 --files "./Images/*.png" --train_data_on_the_fly --validation_data_on_the_fly --early_stopping_frequency 5 --validation './validation/*.png' --output_dir "./trained_model" --weights "./trained_model/best.ckpt"

I am unable to start the training for version 2.3.1. Please help @ChWick

**Comment by ChWick** [comment]
The API's changed massively. Have a look to this issue https://github.com/Calamari-OCR/calamari/issues/252

**Comment by bertsky** [comment]
> The API's changed massively.

And the CLI.

https://github.com/bertsky/hsbcala shows equivalent scripts for C2 and C1 (both training and prediction) next to each other.
